src/components/button/templates/component/style.js

```diff
@@ -132,181 +128,195 @@ export let componentStyle = `
     /* Small */
 
     @media only screen and (min-width : 0px) {
+
         body {
             height: 42px;
         }
 
         body.fundingicons {
             height: 65px;
         }
-    }
-
-    @media only screen and (min-width : 0px) and (min-height: 0px) {
 
         .paypal-button {
-            display: block;
             max-width: 200px;
         }
 
         .paypal-button .paypal-button-content {
-            height: 24px;
-            max-height: 24px;
-            border-radius: 12px;
+            height: 26px;
+            border-radius: 13px;
+        }
+
+        .paypal-button .paypal-button-content,
+        .paypal-button .paypal-button-content .paypal-tagline {
```

Comment:
` .paypal-button .paypal-button-tag-content {}`
Class name in `template.js` is paypal-button-tag-content for the tag line